installers/gcp-marketplace/inventory.ini

```diff
@@ -0,0 +1,104 @@
+localhost ansible_connection=local
+
+[all:vars]
+kubernetes_in_cluster='true'
+create_rbac='true'
+
+pgo_admin_username='admin'
+pgo_admin_password='${OPERATOR_ADMIN_PASSWORD}'
+
+# PGO Admin Role & Permissions
+pgo_admin_role_name='pgoadmin'
+pgo_admin_perms='DeleteNamespace,CreateNamespace,UpdatePgorole,ShowPgorole,DeletePgorole,CreatePgorole,UpdatePgouser,ShowPgouser,DeletePgouser,CreatePgouser,Cat,Ls,ShowNamespace,CreateDump,RestoreDump,ScaleCluster,CreateSchedule,DeleteSchedule,ShowSchedule,DeletePgbouncer,CreatePgbouncer,DeletePgpool,CreatePgpool,Restore,RestorePgbasebackup,ShowSecrets,Reload,ShowConfig,Status,DfCluster,DeleteCluster,ShowCluster,CreateCluster,TestCluster,ShowBackup,DeleteBackup,CreateBackup,Label,Load,CreatePolicy,DeletePolicy,ShowPolicy,ApplyPolicy,ShowWorkflow,ShowPVC,CreateUpgrade,CreateUser,DeleteUser,UpdateUser,ShowUser,Version,CreateFailover,UpdateCluster,CreateBenchmark,ShowBenchmark,DeleteBenchmark,UpdateNamespace'
+
+pgo_installation_name='${OPERATOR_NAME}'
+pgo_operator_namespace='${OPERATOR_NAMESPACE}'
+namespace=''
+
+ccp_image_prefix='docker.io/crunchydata'
+ccp_image_tag='centos7-11.5-2.4.2'
+pgo_image_prefix='docker.io/crunchydata'
+pgo_image_tag='centos7-4.1.0'
+
+pgo_image='${OPERATOR_IMAGE}'
+pgo_event_image='${OPERATOR_IMAGE_EVENT}'
+pgo_apiserver_image='${OPERATOR_IMAGE_API}'
+pgo_scheduler_image='${OPERATOR_IMAGE_SCHEDULER}'
+
+# PGO Client Install
+pgo_client_install='false'
+pgo_client_version='v4.1.0'
```

Comment:
This variable is required even when `pgo_client_install='false'`.

Reply:
We should just leave it for now then.